yapftests/yapf_test.py

```diff
@@ -15,6 +15,7 @@
 """Tests for yapf.yapf."""
 
 import io
+from re import match
```

Comment:
Like you have so polluted namespace anyway... It doesn't do any good, admit it =) I mean specifically in this case.